core/src/main/java/hudson/cli/DeleteNodeCommand.java

```diff
@@ -78,7 +78,7 @@ protected int run() throws Exception {
                 }
 
                 node.toComputer().doDoDelete();
-            } catch (AccessDeniedException e) {
+            } catch (AccessDeniedException2 e) {
```

Comment:
Unfortunately, Jenkins is not very consistent in using its own subclass (`AccessDeniedException2`). Despite I am not sure if good old `AccessDeniedException` can bubble up here somehow, I do not see a reason why to be overly specific when catching exceptions.